docs/developer-docs/latest/setup-deployment-guides/installation/docker.md

```diff
@@ -346,13 +346,12 @@ RUN yarn build
 
 
 FROM node:16-alpine
-RUN apk add vips-dev
-RUN rm -rf /var/cache/apk/*
+RUN apk add --no-cache vips-dev
 ARG NODE_ENV=production
 ENV NODE_ENV=${NODE_ENV}
 WORKDIR /opt/app
 COPY --from=build /opt/node_modules ./node_modules
-ENV PATH /opt/node_modules/.bin:$PATH
+ENV PATH /opt/app/node_modules/.bin:$PATH
```

Comment:
@derrickmehaffy  this will be wrong and break it security.
it should not be changed, as we want to keep it outside the strapi project.